src/manager/RealPrintManager.ts

```diff
@@ -40,31 +42,45 @@ const printJobStateByRealPrintStatus = new Map<
 
 const DefaultPrintJobState = PrintJobState.Unknown
 
+type UUID = ReturnType<typeof uuid>
```

Comment:
I guess I envisioned the use of UUIDs as an implementation detail that didn't belong in the `PrintManager` interface. That would make this a local concern of this file. I could be convinced to make `PrintJob#id` a UUID instead of a `string`, though.